optional_plugins/golang/avocado_golang/__init__.py

```diff
@@ -36,6 +36,30 @@
     GO_BIN = None
 
 
+def find_tests(test_path):
```

Comment:
All these new methods can be marked as static.

Reply:
Well, with this patch they're not methods anymore, and that's actually the purpose of this change... by making it a function, I'll be able to use it in another unrelated class (the resolver implementation).  But, I guess it wasn't easy to understand what I meant here, given that I've not posted (yet) the code that uses it.

I can defer these to a larger PR (with the resolver work), but I though I'd ease the review by posting smaller patches in advance.

Reply:
Thanks for the explanation, this was only thing where I saw the problem. So I will merge that.